m4/vtk.m4

```diff
@@ -119,22 +119,41 @@ AC_DEFUN([CONFIGURE_VTK],
        dnl vtkConfigure.h and vtkVersionMacros.h
        AS_IF([test "x$enablevtk" = "xyes"],
              [
-               dnl If we have the vtkVersionMacros.h (VTK 6.x) find the version there
-               AS_IF([test -r $VTK_INC/vtkVersionMacros.h],
+               AS_IF(
+                     dnl If we have vtkVersionQuick.h (VTK 9.4), look
+                     dnl for the version there.  Note that their
+                     dnl vocabulary has now fixed the "subminor/patch
+                     dnl numbers are not build numbers" problem, by
+                     dnl replacing it with a "subminor/patch numbers
+                     dnl are not epoch numbers" problem.
+                     [test -r $VTK_INC/vtkVersionQuick.h],
+                     [
+                       vtkmajor=`grep "define VTK_MAJOR_VERSION" $VTK_INC/vtkVersionQuick.h | sed -e "s/.*#define VTK_MAJOR_VERSION[ ]*//g"`
+                       vtkminor=`grep "define VTK_MINOR_VERSION" $VTK_INC/vtkVersionQuick.h | sed -e "s/.*#define VTK_MINOR_VERSION[ ]*//g"`
+                       vtkbuild=`grep "define VTK_EPOCH_VERSION" $VTK_INC/vtkVersionQuick.h | sed -e "s/.*#define VTK_EPOCH_VERSION[ ]*//g"`
+                     ],
+                     dnl If we have the vtkVersionMacros.h (VTK 6.x) and have yet to find the version, look there
+                     [test "x$vtkmajor" = "x" && test -r $VTK_INC/vtkVersionMacros.h],
```

Comment:
why the introduction of the `vtkmajor` test? This is the else condition so we didn't set vtkmajor in the if branch. No harm in testing; mostly asking out of curiosity

Reply:
Ah, this is the else condition.  I guess I was thinking about fallthrough (which would cause the valid variables to then get overwritten), but that won't happen.